python/rikai/spark/sql/codegen/sklearn.py

```diff
@@ -16,8 +16,13 @@
 
 import numpy as np
 import pandas as pd
-from pyspark.sql.functions import pandas_udf, PandasUDFType
-from pyspark.sql.types import StructType
+from pyspark.serializers import CloudPickleSerializer
+from pyspark.sql.functions import pandas_udf
+from pyspark.sql.types import BinaryType
+
+__all__ = ["generate_udf"]
+
+_pickler = CloudPickleSerializer()
```

Comment:
nit: how come this is `_pickler` here but `pickler` in pytorch flavor

Reply:
it was separated by a ☕ break ;)  will update the code to make them consistnet.